File: problem/PTAOJ/round2025ReShen/1-5.py
n,m = map(int,input().split())
cnt = 0
ans:str
for _ in range(n):
    s = input()
    if not "qiandao" in s and not "easy" in s:
        m-=1
        if m == -1 :ans = s
if m > -1:
    print("Wo AK le")
else :print(ans)

File: problem/PTAOJ/round2025JveSai/1-5.cpp
#include <bits/stdc++.h>

int main() {
    std::string s;
    std::cin >> s;
    std::vector<int> cnt(26);
    for (auto i: s) cnt[i - 'a']++;
    std::vector<int> w(26);
    for (auto &i: w) std::cin >> i;
    long long ans = 0;
    for (int i = 0; i < 26; i++) {
        ans += w[i] * cnt[i];
    }
    for (int i = 0; i < 26; i++) {
        std::cout << cnt[i] << " \n"[i == 25];
    }
    std::cout << ans << std::endl;
}
